make.js

```diff
@@ -140,7 +141,8 @@ target.web = function() {
         ],
         preprocess: [
             [SRC_DIR + 'crxviewer.html', dest_dir],
-            [SRC_DIR + 'crxviewer.js', dest_dir]
+            [SRC_DIR + 'crxviewer.js', dest_dir],
+            [SRC_DIR + 'lib/crx-to-zip.js', dest_dir + 'lib'] // See https://github.com/Rob--W/crxviewer/issues/26
```

Comment:
The comma is still missing, and "// See http.." should be removed. Once merged, the repository will include references to your pull request and the commit, so there is no need to include the URL.

I suggest to squash all commits and give it a description that includes the original bug report, like this:

```
git reset master
git commit -v -m "Add crx-to-zip.js to preprocessor #26"
```